deploy

```diff
@@ -1,20 +1,36 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This is not portable :) Please use `/usr/bin/env bash` if you really want to use Bash.